UnrealPlugin/README.md

```diff
@@ -218,16 +206,30 @@ Don't forget to connect all the nodes to the Event Init node.
 In the end add the "Ready for Players" to be able to react to the ready signal of Playfab.
 ![Playfab GSDK Full Graph](Documentation/BlueprintFullGraph.png)
 
+## Set the Game Instance class
+    
+After creating a custom game instance class that integrates with the gsdk, you have to configure your project to actually use this newly created game instance class. There are two ways to do this - either through the Unreal Engine editor or by editing DefaultEngine.ini directly. 
+###### In the Unreal Editor
```

Comment:
nit: it's a good practice to have an empty line before a new header

Reply:
https://github.com/chris-gong/gsdk/commit/1ed80bb14872e2db088c4befd86eb107f685b03b